### DISCUSSION TITLE
Sans serif or unmodulated calligraphic/script and Fraktur letters

### DISCUSSION THREAD

**Discussion by apoorvpotnis** [Lisnoti/Lisnoti#4] [General]
Hello, since these fonts are meant to be used in a math/science context, I have a question/feature-request.

Even though the Noto Sans Math fonts were recently made truly sans serif, the calligraphic/script and Fraktur characters are still serif/unmodulated. Creating sans serif or unmodulated versions of them would harmonize the font, in my humble opinion. Are there any plans to create sans serif versions in Lisnoti? If yes, then they can be used for Noto Sans Math as well, which contains a MATH table. Lisnoti fonts are intended to be used in programming contexts, and I recently found out that automated theorem provers, such as Lean, use monospaced fonts, with unfortunately with serif calligraphic/script and Fraktur.

Thank you.

**Comment by TimGord** [comment]
Hi

I’ve experienced similar frustrations in the past trying to find a good sans serif typeface for use in LaTeX. But these days I simply don’t use LaTeX – my maths is all in Markdown (and therefore in whatever Computer Modern-equivalent font is the default in MathJax and KaTeX).

Regarding your specific question, I’m not a font designer – Lisnoti pulls together various existing Noto fonts with a few tweaks by hand for consistency / correction. So I don’t think I’m going to be able to help, sorry.

Finally, maybe I’m misunderstanding, but I wouldn’t expect script and Fraktur glyphs to vary between sans serif and serif (given that they are neither).

Cheers
